tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -773,6 +773,21 @@ def setUpClass(cls):
                                                 org=cls.org,
                                                 loc=cls.loc
                                                 )
+        os = entities.OperatingSystem().search(query={
```

Comment:
in this case we are configuring provisioning with an os and here you search for an os, to exclude any conflict situation and to be sure that the host group and this test case are using the same os and arch we should pass them to  configure_provisioning and if they are are not passed to configure_provisioning then configure_provisioning must search them by it's own mean as it's doing now.

Reply:
I don't want to edit configure_provisioning as it is , used by discovery tests , but yes , i will get the os from configure provisioning and then use it in test case , rather than searching it.

Reply:
In that case the configure_provisioning can find an os RHEL 6 as first element and os of the VM will rhel 6 every where that was not acceptable in my PR https://github.com/SatelliteQE/robottelo/pull/5641 in the same test case